content/plugins/source-map-dev-tool-plugin.md

```diff
@@ -0,0 +1,5 @@
+---
+title: source-map-dev-tool-plugin.md
+---
+
+?> TODO
```

Comment:
Easiest way out - pick up the content from https://webpack.github.io/docs/list-of-plugins.html#sourcemapdevtoolplugin source (markdown) and format to fit.